timeless/restaurants/table_shapes/views.py

```diff
@@ -49,23 +49,27 @@ def create():
 
 @bp.route("/edit/<int:id>", methods=("GET", "POST"))
 def edit(id):
-    """ Edit table shape with id
-    @todo #162:30min Implement edit() method of TableShape model and
-     update template. Use TableShapeForm for that, see how in works in
-     create function. Check and update html template if it's needed.
-    """
-    if request.method == "POST":
-        flash("Edit not yet implemented")
+    """Edit table shape with id"""
+    table = models.TableShape.query.get(id)
+    if not table:
+        return redirect(url_for("tables.list"))
```

Comment:
@krzyk I think we should display ``404`` page, instead of simply redirecting to the list page. Suppose they enter the edit link into the browser directly, then they should see 404 since the page does not actually exist.